### ISSUE TITLE
Update version in dashboard when (won on demand)

### ISSUE THREAD

**Issue by BirdInFire** [tailscale/tailscale#12179]
### What is the issue?

I use on my Mac tailscale as a failsafe when I need internet into a network unknown, the fact is it's rarely the case, and when tailscale update it do not connect the tunnel and the dashboard do not update the fact that I have the new version.

### Steps to reproduce

Use tailscale on a trusted network in "except on" of vpn on demand and update, the version is the last but this version is not updated on the dashboard.

### Are there any recent changes that introduced the issue?

I think it's an issue caused to the fact that those data are send when the tunnel connect, so a change to send them even is disconnected when running a new version for the first time.

### OS version

Mac OS Sonoma 14.5 (23F79)

### Tailscale version

1.66.3

### Other software

N/A

**Comment by agottardo** [comment]
That is correct; the Tailscale app alone does not send data to the coordination server until the VPN actually connects, and that's why you're observing this behavior.

**Comment by awly** [comment]
This appears to be working as intended, the coordination server can't know what the version is for clients that aren't connected.
